WORKING_GROUPS.md

```diff
@@ -14,7 +14,8 @@ think it would benefit from being done as an autonomous project.
 
 If the work defined in a Working Group charter is completed the Working
 Group should be dissolved and the responsibility for governance absorbed
-back in to the CTC.
+back in to the CTC. A Working Group can be dissolved either through consensus 
+of the Working Group membership or normal CTC motion and vote.
```

Comment:
Since we're pretty explicit elsewhere (I think, at least) that the only power the CTC maintains over a WG is to revoke its charter, I think we should use the same language here. I agree the difference is subtle, but consistency will help reduce the likelihood that someone interprets one document as saying one thing and another document as saying something different.